src/views/TheChatbox.vue

```diff
@@ -153,7 +153,9 @@ export default {
       thumbDownFilledUrl: thumbDownFilled,
       thumbButtonClicked: false, // Property to track whether any thumb button is clicked
       copyimgUrl: copyimg, // Original image 
-      copyFilledUrl: copyFilled, // Image  after clicking the button  
+      copyFilledUrl: copyFilled, // Image  after clicking the button
+      votingStates: {}, // Object to store voting states for each answer
+      disableThumbs: {}, // Object to store disabled state for each message  
```

Comment:
Why is there a need of votingSates and disableThumbs objects. 
Simply create add an extra 'vote' property in message objects and render and disable buttons accordingly.